src/cadenza.js

```diff
@@ -785,6 +807,15 @@ function createParams({
       webApplicationLink: webApplication.externalLinkId,
       webApplicationLinkRepository: webApplication.repositoryName,
     }),
+    ...(filter &&
+      Object.entries(filter).reduce(
+        (filterVarParameters, [varKey, varValue]) => ({
+          ...filterVarParameters,
+          [`filter.${varKey}`]:
+            varValue instanceof Date ? varValue.toISOString() : varValue,
```

Comment:
The individual values must be passed to JSON.stringify(), so that there are quotes around the (date) strings.

Reply:
Fixed.